### PR TITLE
Fix flake8 warnings and ignore VNE001 for Python samples

### PR DESCRIPTION
**Pull request by jiwaszki**
### Details:
 - Ignore the rule of flake8-variables-names "single letter variable names" which is not applicable in samples.
 - Fix other minor issues in other files.

### Tickets:
 - N/A

### CODE REVIEW

**Comment by Wovchena** [comment]
Why is it not applicable for samples? I would prefer to follow this rule. Though I leave the final derision up to you

**Comment by mlukasze** [comment]
hey @Wovchena 
if you have bandwidth please prepare a fix and enable this rule again after it.
for now we have to unblock CI

**Review by mlukasze — APPROVED** [review]

**Comment by jiwaszki** [comment]
> Why is it not applicable for samples? I would prefer to follow this rule. Though I leave the final derision up to you

Generally, you have a really good preference! I would follow it as well.

However all reported lines are "shape deconstructions" like `n, h, w, c = input_tensor.shape` -- should it really be changed to `batch, height, width, channels = ...` every time? This naming convention using letters is well-known, especially when talking about the data layout.

This is why I only change this rule for samples, not for the whole project where separate lines can be suspended on demand. I think `code code code # noqa: VNE001` is even worse for samples, so "global" suspension works better.

**Comment by Wovchena** [comment]
Ok. As I said, I leave the final derision up to you

**Review by p-wysocki — APPROVED** [review]

**Review by praasz — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

File: samples/python/setup.cfg [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # D103 - Missing docstring in public function
```
Comment (rkazants) [inline samples/python/setup.cfg:5]: please add explanation of this error `VNE001`

**Comment by ilya-lavrenov** [comment]
Maybe we need to create a ticket to properly fix the issue?

**Comment by mlukasze** [comment]
> Maybe we need to create a ticket to properly fix the issue?

I would leave this decision to be made later, to give Vladimir & Jan chance to discuss pros & cons (as above), incl proper description of exception, now let's only unblock CIs

**Comment by jiwaszki** [comment]
> Maybe we need to create a ticket to properly fix the issue?

@ilya-lavrenov if you want to, sure go ahead. But if you ask me this change is not applicable in samples where "shape deconstructions" like `n, h, w, c = input_tensor.shape` is some general, widely used terminology.

**Comment by ilya-lavrenov** [comment]
> > Maybe we need to create a ticket to properly fix the issue?
> 
> @ilya-lavrenov if you want to, sure go ahead. But if you ask me this change is not applicable in samples where "shape deconstructions" like `n, h, w, c = input_tensor.shape` is some general, widely used terminology.

I agree about terminology, but maybe we need to suppress specific places instead of putting it into a "global config"?

**Review by rkazants — APPROVED** [review]

**Review by mbencer — APPROVED** [review]

**Pull request merged**